modules/prepare-ocp-for-bare-metal-builds.adoc

```diff
@@ -0,0 +1,199 @@
+:_content-type: PROCEDURE
+[id="prepare-ocp-for-bare-metal-builds"]
+= Configuring bare metal builds for {productname-ocp}
+
+Use the following procedure to configure _bare metal builds_ for {productname-ocp}.
+
+[NOTE]
+====
+If you are using the {productname} Operator on {ocp} with a managed `route` component in your `QuayRegistry` CRD, see "{productname-ocp} _builds_ limitations with self-managed _routes_".
+====
+
+.Prerequisites 
+
+* You have an {ocp} cluster provisioned with the {productname} Operator running.
+* You have set the `tls` component to `unmanaged` and uploaded custom SSL/TLS certificates to the {productname} Operator. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_quay/{producty}/html-single/securing_red_hat_quay/index#ssl-tls-quay-overview[SSL and TLS for {productname}].
```

Comment:
Can't access the link `https://docs.redhat.com/en/documentation/red_hat_quay/3.13/html-single/securing_red_hat_quay/index#ssl-tls-quay-overview`
Got below error message
<img width="1701" alt="Screenshot 2024-10-03 at 23 39 54" src="https://github.com/user-attachments/assets/1d3ed463-9dc4-48c3-bf06-d2caa67635b5">

Reply:
Thanks! This won't render until 3.13 is live. That content/book does not exist yet (it's ready for publication though).